domain/src/main/kotlin/no/nav/su/se/bakover/domain/Behandling.kt

```diff
@@ -153,11 +157,36 @@ data class Behandling(
     }
 
     private fun nyTilstand(target: Tilstand): Tilstand {
+        // Ønsker å kunne vise hvor mange behandlinger vi har i de forskjellige tilstandene. Ved restart kan gaugen bli negativ, men dersom man klarer å summe de vil vi kanskje få lurt det til.
+        incrementStatusGauge(target.status)
+        decrementStatusGauge(status)
+        incrementStatusCounter(target.status)
         status = target.status
         tilstand = resolve(status)
         return tilstand
     }
 
+    private val behandlingstatuserGauge = BehandlingsStatus.values().map {
+        it to AtomicInteger(0)
+    }.toMap()
+
+    private fun incrementStatusGauge(status: BehandlingsStatus) = Gauge.builder("førstegangsbehandling_gauge") {
+        behandlingstatuserGauge.getValue(status).incrementAndGet()
+    }
+        .tag("type", status.name)
+        .register(Metrics.globalRegistry)
+
+    private fun decrementStatusGauge(status: BehandlingsStatus) = Gauge.builder("førstegangsbehandling_gauge") {
+        behandlingstatuserGauge.getValue(status).decrementAndGet()
+    }
+        .tag("type", status.name)
+        .register(Metrics.globalRegistry)
+
+    private fun incrementStatusCounter(status: BehandlingsStatus) = Counter.builder("førstegangsbehandling")
+        .tag("type", status.name)
+        .register(Metrics.globalRegistry)
```

Comment:
Funker disse for grafana/prometheus hvis vi ikke bruker "PrometheusRegistry"? Hva er evt forskjellen? Tror for øvrig vi allerede har installert en slik i "SuMetrics".

Reply:
Det funker ja! Det er i praksis det som allerede var brukt, bare implisitt ved å bruke deres statiske count(name:String) funksjon.

Reply:
Pgr. dette:
```
 public static void addRegistry(MeterRegistry registry) {
        globalRegistry.add(registry);
    }
```